airbrakes/data_handling/apogee_predictor.py

```diff
@@ -249,17 +278,35 @@ def _prediction_loop(self) -> None:
 
             if len(self._accelerations) - last_run_length >= APOGEE_PREDICTION_MIN_PACKETS:
                 self._cumulative_time_differences = np.cumsum(self._time_differences)
-                # We only want to keep curve fitting if the curve fit hasn't converged yet
-                if not self._has_apogee_converged:
-                    curve_coefficients = self._curve_fit()
-                    self._update_prediction_lookup_table(curve_coefficients)
+
+                # Placeholder values for the curve coefficients and apogee
+                curve_coefficients = -1, -1
+                apogee = -1
```

Comment:
let's use `self.apogee` and `self.curve_coefficients` and initialize them to `None`